build/windows-script.ps1

```diff
@@ -0,0 +1,30 @@
+# Get the arch and dir
+param([String]$arch="x86_64")
+
+$NAME = "VapourSynth"
+$PY_DIR = "py-dir"
+$VS_DIR = "vs-dir"
+
+If ($arch -eq "i686") {
+        $SUFFIX = 32
+        $PYTHON_PKG = "python-3.7.5-embed-win32.zip"
+} Else {
+        $SUFFIX = 64
+        $PYTHON_PKG = "python-3.7.5-embed-amd64.zip"
+}
+
+# Download Python embeddable and VapourSynth portable
+$VS_PATH = "https://github.com/vapoursynth/vapoursynth/releases/download/R47.2"
+curl -LO "https://www.python.org/ftp/python/3.7.5/$PYTHON_PKG"
+curl -LJO "$VS_PATH/VapourSynth$SUFFIX-Portable-R47.2.7z"
```

Comment:
Why is `-J` needed here?

Reply:
Oh, my bad. Forgot to remove it. Thanks!